app/livechat/client/stylesheets/livechat.less

```diff
@@ -34,7 +34,8 @@
 	}
 }
 
-.livechat-code {
+.livechat-code,
+.old-livechat-code {
```

Comment:
please use the same class for styling (you don't need to create a new one `.old-livechat-code`)